guides/common/modules/con_smartproxy-overview.adoc

```diff
@@ -0,0 +1,12 @@
+[id="{SmartProxy}-Overview_{context}"]
+= {SmartProxy} overview
+
+With {SmartProxyServers}, you can extend the reach and scalability of your {Project} deployment.
+{SmartProxyServers} provide the following functionalities in a {ProjectName} deployment:
+
+* Mirroring content from {ProjectServer} to establish content sources in various geographical locations.
```

Comment:
```suggestion
* Mirroring content from {ProjectServer} to establish content sources in various geographical and logical locations.
```

maybe "logical" for multiple networks (DMZs) in the same rack/data center?

Reply:
Applied.